src/Utilities/WellKnownTypes.cs

```diff
@@ -7,16 +7,124 @@ namespace Analyzer.Utilities
 {
     internal static class WellKnownTypes
     {
+        public const string SystemWebHttpRequest = "System.Web.HttpRequest";
+        public const string SystemDataIDataAdapter= "System.Data.IDataAdapter";
+        public const string SystemDataIDbCommand = "System.Data.IDbCommand";
         public const string SystemException = "System.Exception";
         public const string SystemDiagnosticContractsContract = "System.Diagnostics.Contracts.Contract";
         public const string SystemIDisposable = "System.IDisposable";
         public const string SystemThreadingMonitor = "System.Threading.Monitor";
         public const string SystemThreadingTasksTask = "System.Threading.Tasks.Task";
         public const string SystemCollectionsICollection = "System.Collections.ICollection";
-        public const string SystemCollectionsGenericICollection1 = "System.Collections.Generic.ICollection`1";
-        public const string SystemCollectionsGenericIReadOnlyCollection1 = "System.Collections.Generic.IReadOnlyCollection`1";
         public const string SystemRuntimeSerializationSerializationInfo = "System.Runtime.Serialization.SerializationInfo";
```

Comment:
I presume most of these will go away once we have a configuration story for these analyzers? Sorry to have inflicted this pain on you. I will work on the configuration story next to avoid more such exercises for you.

Reply:
They certainly can.  No worries.  I was kinda wavering between hardcoding everything and configurationing everything before anyway.